internal/slicer/slicer_test.go

```diff
@@ -16,9 +16,20 @@ import (
 	"github.com/canonical/chisel/internal/testutil"
 )
 
+var (
+	Reg = testutil.Reg
+	Dir = testutil.Dir
+	Lnk = testutil.Lnk
+)
+
+type testPackage struct {
+	content []byte
+}
+
 type slicerTest struct {
 	summary string
 	arch    string
+	pkgs    map[string]map[string]testPackage
```

Comment:
Should this be named `pkgs`? It seems to be a map of archives. Inside those archives are maps of packages. How about `archives`?

And how about breaking down the `map[string]map[string]testPackage` type to `map[string]testArchive`?

Reply:
We discussed it out of band already. It is mapping from archive names to mapping of package names to package test package structures. I'm not sure `archives` improves the matter.

Regarding your second suggestion, this would lead to additional indentation in test case definition required by the `testArchive` structure.

Reply:
> It is mapping from archive names to mapping of package names to package test package structures. I'm not sure archives improves the matters.

I'm not sure either, but that sentence itself might be a hint that something could be improved. :)